editor/plugins/tiles/tiles_editor_plugin.h

```diff
@@ -126,14 +126,42 @@ class TilesEditorPlugin : public EditorPlugin {
 	void set_sorting_option(int p_option);
 	List<int> get_sorted_sources(const Ref<TileSet> p_tile_set) const;
 
+	void edit(Object *p_object);
+	void make_visible(bool p_visible);
+
+	static void draw_selection_rect(CanvasItem *p_ci, const Rect2 &p_rect, const Color &p_color = Color(1.0, 1.0, 1.0));
+
+	TilesEditorUtils();
+	~TilesEditorUtils();
+};
+
+class TileMapHandlerPlugin : public EditorPlugin {
```

Comment:
I think we should go the other way around and rename `TileMapEditorPlugin` (the one in the current codebase) to something else. Something like `TileMapEditorHandler`, since "Plugin" is a bit misleading there.

And then, `EditorPlugins` should be named `TileMapEditorPlugin`  and `TileSetEditorPlugin`